pki/util.go

```diff
@@ -248,6 +248,7 @@ func getControlCertKeys() []string {
 		KubeSchedulerCertName,
 		KubeProxyCertName,
 		KubeNodeCertName,
+		KubeAdminCertName,
```

Comment:
why do you deploy the kubeadmin certs on the control nodes?

Reply:
Removed, deployment is now controller by Rancher server template.